core/domain/skill_services.py

```diff
@@ -112,18 +119,25 @@ def _get_skill_summaries_in_batches(
 
 
 def get_filtered_skill_summaries(
-        num_skills_to_fetch, status, classroom_name, keywords,
-        sort_by, urlsafe_start_cursor):
+    num_skills_to_fetch: int,
+    status: Optional[str],
```

Comment:
Why are `status` and `classroom_name` Optional?

Reply:
In many test cases we are providing None, but apart from test cases If we see the functions that are called inside this method we can see that there we are handling the case of None. 
Eg:
https://github.com/oppia/oppia/blob/cc8f6d73d492d77aada65e73d7f94d258746185d/core/domain/skill_services.py#L230 

Also `_filter_skills_by_status` is only used here.